### PR TITLE
[Bug Fix] - Autofire attacking yourself

### PR DESCRIPTION
**Pull request by splose**
Fix being able to attack yourself with autofire if Combat:MinRangedAttackDist == 0

### CODE REVIEW

File: zone/client_process.cpp [hunk 1]
```diff
@@ -302,6 +302,10 @@ bool Client::Process() {
 		}
 
 		if (AutoFireEnabled()) {
+			if (this->GetTarget() == this) {
```
Comment (Kinglykrab) [inline zone/client_process.cpp:305]: Shouldn’t need this-> here.

File: zone/client_packet.cpp [hunk 2]
```diff
@@ -3403,6 +3403,13 @@ void Client::Handle_OP_AutoFire(const EQApplicationPacket *app)
 		DumpPacket(app);
 		return;
 	}
+
+	if (this->GetTarget() == this) {
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:3407]: Shouldn’t need this-> here.

**Comment by Kinglykrab** [comment]
Looks good, small comments.

File: zone/client_packet.cpp [hunk 3]
```diff
@@ -3403,6 +3403,13 @@ void Client::Handle_OP_AutoFire(const EQApplicationPacket *app)
 		DumpPacket(app);
 		return;
 	}
+
+	if (GetTarget() == this) {
+		this->MessageString(Chat::TooFarAway, TRY_ATTACKING_SOMEONE);
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:3408]: Don't need this-> here.

Comment (splose) [reply zone/client_packet.cpp:3408]: yes you do lol

**Review by Kinglykrab — APPROVED** [review]

**Pull request merged**